File: Basic Challenges/Factorial Calculator/Factorial.java
public class Factorial{

    public static long factorial(long n){
		if(n>0){
			return factorial(n-1)*n;
		}
		else
		{
			return 1;
		}
	}

    public static void main(String []args)
    {
        //Calculate factorial number using a recursive function
        factorial(6);
    }
}

File: Basic Challenges/Factorial Calculator/factorial.py

n=int(input("number:"))
p=1
for i in range(2,n+1):
    p*=i
print("output:",p)
=======
def factorial(num):
    if num==0:
        return 1
    else:
        return num*factorial(num-1)
num=int(input("Enter your number here:"))
factorial(num)
